pkg/provision/storage/storage.go

```diff
@@ -719,7 +720,7 @@ func (b *BoltStorage) Capacity() (cap gridtypes.Capacity, active []gridtypes.Dep
 		}
 	}
 
-	return cap, active, nil
+	return cap, active, workloads, nil
```

Comment:
That's duplicate data because the deployments include all the workloads anyway. So may be instead return the `number` of active workloads instead of list of active workloads.